### ISSUE TITLE
[Feature Request] Max IVs Searching

### ISSUE THREAD

**Issue by SonicBlader** [wwwwwwzx/SMEncounterRNGTool#43]
Hello!

Well when RNGing certain pokemon like Mystery Gift Pokemon, it is hard to get more than 4 max IVs without waiting for a loooong time. Finding those 4 max IVs itself is a pain as we need to keep searching for various spreads as they are in random stats. So I feel that it would be better to have a feature which allows us to select the minimum number of perfect IV in a stat which we want in the spread. Like if I select 5, I will get only frames which have atleast 5 Max IVs. This will greatly help in searching for pokemon!

Thanks! :D

**Comment by wwwwwwzx** [comment]
this feature is feasible. but i have run out of space for new UI control :(

**Comment by SonicBlader** [comment]
Thanks for the reply!
 I just took a look at the latest code and well its pretty much out of space. But I could wriggle out a little bit of space for it. Here is my crude attempt at it - http://oi66.tinypic.com/fcmfs5.jpg
That panel can be renamed as Misc Settings/Controls and it can have the SOS related(?) option on one side and the max IVs on the other side.

Another option would be to resize the TimeResult listbox as currently it is quite big. Also does it display an data from the second line? As far as I have tried it I haven't been able to get any output in the latter lines. The first line gets replaced with the updated results. So a fair amount of space can be saved there.
